spmd/tensor/parallel/api.py

```diff
@@ -12,7 +12,12 @@
     Placement,
 )
 from spmd.tensor.parallel import TensorParallelMultiheadAttention
-from spmd.tensor.parallel.style import ParallelStyle, PairwiseParallel
+from spmd.tensor.parallel.style import (
+    ParallelStyle,
+    RowwiseParallel,
+    ColwiseParallel,
+    PairwiseParallel,
```

Comment:
Why do you mark the comment as resolved? Looks like this is not sorted? And yes we didn't do that previously and this needs to be corrected. Thanks!

Reply:
just my confusion: do we usually force `import` sorted or leave it in any order?